scripts/Muon/GUI/Common/seq_fitting_tab_widget/QSequentialTableModel.py

```diff
@@ -16,11 +16,11 @@
 default_chi_squared = 0.0
 
 
-class SequentialTableModel(QAbstractTableModel):
+class QSequentialTableModel(QAbstractTableModel):
```

Comment:
could we have a doc string with a link to the documentation that talks about Qt models?